package.json

```diff
@@ -13,7 +13,9 @@
     "type": "git",
     "url": "https://github.com/MadLittleMods/postcss-css-variables.git"
   },
+  "preinstall": "npm install escape-string-regexp extend match-recursive postcss",
```

Comment:
Why was this added?

Reply:
We faced a particular problem with missing packages, o I had to use `preinstall` instruction to force add the missing packages. I removed it now.